roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -188,7 +221,8 @@ class Namespace(object):
             yield int(pid)
 
     def interfaces(self):
-        return Interface.enumerate(namespace=self.namespace)
+        return [x for x in Interface.enumerate(namespace=self.namespace) \
```

Comment:
This is now neutron_interfaces() since it filters the interfaces in the namespace, or better yet move the filtering logic into the calling method.